src/app/static/src/app/store/filteredData/filteredData.ts

```diff
@@ -5,7 +5,7 @@ import { getters } from './getters';
 import { RootState} from "../../root";
 import { FilterOption } from "../../generated";
 
-export enum DataType { ANC, Survey, Program }
+export enum DataType { ANC, Program, Survey }
```

Comment:
Changed these to be alphabetical so we can easily know which enum value corresponds to which number (see above comment)